make_obsparams.py

```diff
@@ -121,9 +172,22 @@ def cli():
     show_default=True,
     help="Split simulation into a number of time chunks",
 )
-def h4c_cli(freq_range, freqs, sky_model, chunks):
+def h4c_cli(layout, ants, freq_range, freqs, sky_model, chunks):
```

Comment:
Rename this function to `cli`.

Reply:
I renamed to `make_obsparams` which seems more informative than simply `cli`